habitat-lab/habitat/tasks/rearrange/multi_task/pddl_domain.py

```diff
@@ -528,22 +611,7 @@ def solution(self):
 
     @property
     def all_entities(self) -> Dict[str, PddlEntity]:
```

Comment:
Can't you just inherit from the PDDLDomain class ?

Reply:
What do you mean? The PDDLDomain class provides some base entities and then PDDLProblem adds more entities to this dict.